filer/utils/files.py

```diff
@@ -5,11 +5,14 @@
 from django.core.files.uploadedfile import SimpleUploadedFile
 from filer.settings import FILER_FILE_MODELS
 from filer.utils.loader import load_object
+import os.path
 
 class UploadException(Exception):
     pass
 
-
+def file_exists_on_filesystem(file_obj):
+    return os.path.exists(file_obj.path)
```

Comment:
Shouldn't this be storage backend agnostic? `os.path.exists` wouldn't work with S3

`django.core.files.storage.Storage` has an `exists` method which I think you should be able to use.